src/editorwidget/plugin.ts

```diff
@@ -80,7 +84,9 @@ const cmdIds = {
   matchBrackets: 'editor:match-brackets',
   vimMode: 'editor:vim-mode',
   closeAll: 'editor:close-all',
-  changeTheme: 'editor:change-theme'
+  changeTheme: 'editor:change-theme',
+  startConsole: 'editor:start-console',
+  runCode: 'editor:run-code'
```

Comment:
Let's decide is we are going to use "run" or "execute" in the UI and commands and then use the word consistently.